ios/ElectrodeApiImpl/APIImpls/Navigation/MiniAppNavViewController.swift

```diff
@@ -22,11 +22,13 @@ open class MiniAppNavViewController: UIViewController, ENNavigationProtocol {
     public var finishedCallback: MiniAppFinishedCallback?
     public var finish: Payload?
     var delegate: ENNavigationDelegate?
+    var viewIdentifier: String
```

Comment:
Can we move this to the `delegate` instead? That would allow us to use the id for other events in the future as well.

Reply:
This id is associated with the viewController, just like miniappName and properties. I think it's better to put it here